easybatch-integration/easybatch-apache-commons-csv/src/main/java/org/easybatch/integration/apache/common/csv/ApacheCommonCsvRecordMapper.java

```diff
@@ -40,7 +43,11 @@
     private ObjectMapper<T> objectMapper;
 
     public ApacheCommonCsvRecordMapper(Class<? extends T> recordClass) {
-        this.objectMapper = new ObjectMapper<T>(recordClass);
+        this(recordClass, true);
+    }
+
+    public ApacheCommonCsvRecordMapper(Class<? extends T> recordClass, boolean emptyToNull) {
```

Comment:
We need to add this parameter in other record mappers that use the ObjectMapper (Delimited, Fixed-Length and Jdbc). For those two reasons, I would prefer implementing the "emptyToNull" enhancement in type converters rather than the ObjectMapper. Do you agree?